front/app/src/Redux/store/index.ts

```diff
@@ -12,23 +12,37 @@ import
 	REGISTER
 } from "redux-persist";
 import storage from "redux-persist/lib/storage";
-import serverSlice from "./server-slice";
 
-const	persistConfig = {
+import	serverSlice from "./server-slice";
+import	anonymousUserSlice from "./anonymousUser-slice";
+
+const	persistConfigController = {
 	key: "root",
 	// version: 2,
 	storage
 };
 
+const	persistConfigAnonymousUser = {
+	key: "anonymousUser",
+	// version: 2,
```

Comment:
is version essential ?

Reply:
version will be essential yes, just not apply for now